example/sl-prediction/configs/collins-roguev.json

```diff
@@ -11,13 +12,14 @@
 	"name": "sp",
 	"ppi": "../../data/ppi/biogrid/sp/sp-biogrid.v3.4.157-ppi-std.tsv",
 	"gi": "../../data/gi/roguev/sp/roguev-sp-emap-gis-std.tsv",
+    "lambda": 0.05,
 	"homologs": {
 	    "A": "../../data/homologs/sc-sp/sp-sc-homologs.txt"
 	}
     },
     "n_landmarks": 400,
     "dataset_name": "collins-roguev",
-    "output_dir": "/scratch0/mdml/HANDL/example/sl-prediction/output",
+    "output_dir": "/scratch0/jfan03/HANDL/example/sl-prediction/output",
```

Comment:
We should probably exclude these from the config files in the future.

Reply:
Agreed.